src/coreclr/vm/codeman.cpp

```diff
@@ -4682,6 +4620,71 @@ PTR_Module ExecutionManager::FindModuleForGCRefMap(TADDR currentData)
     return dac_cast<PTR_Module>(pRS->pHeapListOrZapModule);
 }
 
+int ExecutionManager::FindRangeSectionHandleHelper(TADDR addr)
+{
+    CONTRACTL {
+        NOTHROW;
+        HOST_NOCALLS;
+        GC_NOTRIGGER;
+        SUPPORTS_DAC;
+    } CONTRACTL_END;
+
+    _ASSERTE(m_RangeSectionHandleArray != nullptr);
+
+    if ((int)m_RangeSectionArraySize == 0)
+    {
+        return -1;
```

Comment:
It is not an invalid index but mirrored around zero index-to-add-into.

Reply:
Call it some other way, but it should be defined somewhere instead of constant in code. -1 constants are used all over the place, and it makes hard to make changes to code, especially if -1 might mean different things

Reply:
No, it is not and should not be a named entity, it is as much distinct as zero index in an regular array relative to other indexes: we do not type arr[STARTING_INDEX], just arr[0].

Reply:
I disagree on this. -1 is not an index directly, it is some entity that you later convert to some index. So, name it. Also, you anyway need to add api that I mentioned below, it can help here too:
```
int getNegativeValueFromIndex(int idx);
int getIndexFromNegativeValue(int value);
```